content/en/apps/concepts/navigation.md

```diff
@@ -36,9 +36,27 @@ Tap the menu icon in the upper right corner of the header to access other pages,
+Data synchronization is important for [offline users]({{< relref "apps/concepts/users" >}}). These users keep a copy of the data they have access to on their device. They can work from their device while disconnected from the internet (offline), by reading from and writing to their copy of the data. “Sync” (synchronization) is when data on the device is made to match the data on the server and requires an internet connection. The CHT app monitors the online status and attempts sync accordingly.
 
-At the bottom of the menu is a notification which provides important information about data synchronisation.
+#### Replication Types
+
+Synchronization consists of upward replication and downward replication. 
+- **Upward Replication**: Uploading all new or updated data from the device to the server. It includes a retry mechanism for handling larger data batches, ensuring a robust and reliable upload process.
+- **Downward Replication**: Downloading new or updated data from the server to the device. Downward replication may include the download of software updates to the CHT app when available.
+
+The CHT application manages data synchronization across two types of databases: the main application database and a meta database.
+
+- **Main Application Database (`medic`)**: The main database that stores the primary data used by the application. It includes contacts, reports, messages, and other critical documents necessary for the core functionality of the application. This database is synchronized continuously to reflect changes to the application, such as new contact creations. Each user stores a subset of the main database which includes only the documents they're allowed to view.
+- **User-Metadata Database(`medic-user-{username}-meta`)**: Each user has a dedicated database that stores operational metadata, including [telemetry data]({{< relref "apps/guides/performance/telemetry" >}}) and error messages. Synchronization occurs at predefined intervals to ensure up-to-date monitoring and analysis.
+
+The CHT application has two different Sync Intervals: 
+- **Regular Sync**: Occurs every 5 minutes. It replicates application data, including user-generated content and any changes to existing documents.
+- **Metadata Sync**: Occurs every 30 minutes, focusing on operational metadata rather than user-generated content. This ensures that telemetry data and system logs are regularly synchronized, providing insights into system performance and user activities
```

Comment:
We should keep the names of the two databases consistent rather than using "regular" here. Suggestion:

```suggestion
The two databases sync on different schedules: the main application database syncs every 5 minutes, while the user metadata database syncs every 30 minutes.
```

Reply:
I deleted the content all-together because the sync interval is mentioned below in the Synchronization Triggers 

`- **Periodic Sync**: The application performs regular checks and attempts to synchronize. The main application database syncs every 5 minutes, while the user metadata database syncs every 30 minutes.`